doc/man3/OPENSSL_fork_prepare.pod

```diff
@@ -11,6 +11,10 @@ OPENSSL_fork_child
 
  #include <openssl/crypto.h>
 
+Deprecated since OpenSSL 3.0.0, can be hidden entirely by defining
+B<OPENSSL_API_COMPAT> with a suitable version value, see
+L<openssl_user_macros(7)>:
+
  void OPENSSL_fork_prepare(void);
  void OPENSSL_fork_parent(void);
  void OPENSSL_fork_child(void);
```

Comment:
Sorry - missed this before. Could you add some text at the start of the description saying that these functions are deprecated and that there is no need for a replacement (similar to the CHANGES.md entry).

Reply:
Have a look at the updated comment now.

Reply:
Sorry, I meant add the text at the beginning of the "DESCRIPTION" section below!

Reply:
Perhaps 342c938 is better.